views/layer/vector.html

```diff
@@ -2,7 +2,7 @@
   <div class="card">
     <div class="btn btn-secondary btn-block btn-sm" title="Style controls for the layer(s)"
         data-toggle="collapse" data-target="#layer-style"
-        ng-click="vector.setOpenSection('#layer-style')">
+        ng-click="vector.setOpenSection('#layer-style')"></div>
```

Comment:
Was this accidental? Line 7 closes this div.

Reply:
Yes it was. Amending this now.